administrator/language/de-DE/com_cpanel.ini

```diff
@@ -16,7 +16,7 @@ COM_CPANEL_MESSAGES_BODY_NOCLOSE="Es gibt wichtige Nachinstallationshinweise. Bi
 COM_CPANEL_MESSAGES_REVIEW="Hinweise anzeigen"
 COM_CPANEL_MESSAGES_TITLE="Es gibt Nachinstallationshinweise"
-COM_CPANEL_MSG_ADDNOSNIFF_BODY="<p>Seit Version 3.9.3 wird Joomla mit zusätzlichen Sicherheitsmaßnahmen in den Standarddateien <strong>htaccess.txt</strong> und <strong>web.config.txt</strong> ausgeliefert. Diese Maßnahmen deaktivieren die sogenannte MIME-Sniffing-Funktion in Webbrowsern. Das Sniffing führt zu speziellen Angriffsvektoren, mit denen Skripte in normalerweise harmlosen Dateiformaten (z.B. Bildern) ausgeführt werden, was zu Cross-Site-Scripting-Schwachstellen führt.</p><p> Das Joomla! Sicherheitsteam (JSST) <strong>empfiehlt dringend</strong>, die notwendigen Änderungen manuell auf bestehende <strong>.htaccess</strong> oder <strong>web.config</strong> Dateien anzuwenden, da diese Dateien nicht automatisch aktualisiert werden können.</p><p><strong>Änderungen in der .htaccess</strong><br />Folgende Zeilen einfügen vor '## Mod_rewrite in use.':</p><pre>&lt;IfModule mod_headers.c&gt;\nHeader always set X-Content-Type-Options 'nosniff'\n&lt;/IfModule&gt;</pre><p><strong>Änderungen in der web.config</strong><br />Folgende Zeilen einfügen direkt nach '&lt;/rewrite&gt;':</p><pre>&lt;httpProtocol&gt;\n  &lt;customHeaders&gt;\n    &lt;add name='X-Content-Type-Options' value='nosniff' /&gt;\n  &lt;/customHeaders&gt\n&lt;/httpProtocol&gt;</pre>" ; Translators: Don't touch the code part in the following message, Starting with ## Mod_rewrite ...
+COM_CPANEL_MSG_ADDNOSNIFF_BODY="<p>Seit Version 3.9.3 wird Joomla mit zusätzlichen Sicherheitsmaßnahmen in den Standarddateien <strong>htaccess.txt</strong> und <strong>web.config.txt</strong> ausgeliefert. Diese Maßnahmen deaktivieren die sogenannte MIME-Sniffing-Funktion in Webbrowsern. Das Sniffing führt zu speziellen Angriffsvektoren, mit denen Skripte in normalerweise harmlosen Dateiformaten (z.&#8239;B. Bildern) ausgeführt werden, was zu Cross-Site-Scripting-Schwachstellen führt.</p><p> Das Joomla! Sicherheitsteam (JSST) <strong>empfiehlt dringend</strong>, die notwendigen Änderungen manuell auf bestehende <strong>.htaccess</strong> oder <strong>web.config</strong> Dateien anzuwenden, da diese Dateien nicht automatisch aktualisiert werden können.</p><p><strong>Änderungen in der .htaccess</strong><br />Folgende Zeilen einfügen vor '## Mod_rewrite in use.':</p><pre>&lt;IfModule mod_headers.c&gt;\nHeader always set X-Content-Type-Options 'nosniff'\n&lt;/IfModule&gt;</pre><p><strong>Änderungen in der web.config</strong><br />Folgende Zeilen einfügen direkt nach '&lt;/rewrite&gt;':</p><pre>&lt;httpProtocol&gt;\n  &lt;customHeaders&gt;\n    &lt;add name='X-Content-Type-Options' value='nosniff' /&gt;\n  &lt;/customHeaders&gt\n&lt;/httpProtocol&gt;</pre>" ; Translators: Don't touch the code part in the following message, Starting with ## Mod_rewrite ...
```

Comment:
Wird glaub in J4.x nicht mehr verwendet. Ist nen Nachinstallationshinweis aus J3.x

Reply:
Der String ist ja weiterhin drin, dann sollte der. auch gfixed werden.

Reply:
Ja klar. Was ich damit eigentlich ausdrücken wollte war, dass ich hier nichts "getestet" habe.